pages/cookbook/jettons.mdx

```diff
@@ -2,9 +2,38 @@
 
 import { Callout } from 'nextra-theme-docs';
 
+## Overview
+
 This page lists common examples of working with [jettons](https://docs.ton.org/develop/dapps/asset-processing/jettons).
 
-## Accepting jetton transfer
+Jettons are token standards on the TON (The Open Network) blockchain, designed for creating fungible tokens (similar to ERC-20 on Ethereum) with a decentralized approach. They are implemented as a pair of smart contracts, typically consisting of two core components:
+
+* Jetton Master Contract (Jetton Master)
+* Jetton Wallet Contract (Jetton Wallet)
+
+These contracts interact with each other to manage token supply, distribution, transfers, and other operations related to the jetton.
+
+### Jetton Master Contract
+
+The Jetton Master Contract serves as the central entity for a specific token. It maintains critical information about the jetton itself and facilitates the creation of individual wallets for token holders. Key responsibilities and data stored in the Jetton Master Contract include:
+
+* Jetton Metadata: Information such as the token's name, symbol, total supply, and decimals.
+* Jetton Wallet Creation: The Jetton Master can generate individual Jetton Wallet contracts for token holders.
+* Minting and Burning: When new jettons are minted (created), the Jetton Master handles the creation process and distributes them to the relevant wallets. Similarly, it manages burning (destruction) of tokens when required.
+* Supply Management: The Jetton Master keeps track of the total supply of the token, ensuring proper accounting for all issued jettons.
+
+### Jetton Wallet Contract
```

Comment:
I think it would be good to explain here that wallets usually communicate between each other without any activity from master, so master contract doesn't create a bottleneck